mappings/net/minecraft/world/WorldEvents.mapping

```diff
@@ -205,7 +205,7 @@ CLASS net/minecraft/unmapped/C_kkbhcwdu net/minecraft/world/WorldEvents
 	FIELD f_ptnwublh TRAVEL_THROUGH_PORTAL I
 		COMMENT A Portal is traveled through.
 		COMMENT <br>Plays the portal travel sound event directly through the client's sound manager.
-		COMMENT <p>Called by {@link net.minecraft.server.network.ServerPlayerEntity#moveToWorld(net.minecraft.entity.Entity$C_bfxofopn) ServerPlayerEntity#moveToWorld}
+		COMMENT <p>Called by {@link net.minecraft.unmapped.C_zbvyjshu#m_ciosvisu(net.minecraft.entity.Entity) C_zbvyjshu#m_ciosvisu}
```

Comment:
Was this a mistake?

Reply:
No, the thing no longer exists and was replaced by an unmapped class